plugins/inputs/modbus/modbus_test.go

```diff
@@ -152,6 +162,27 @@ func TestHoldingRegisters(t *testing.T) {
 			write:     []byte{0x08, 0x98},
 			read:      float64(220),
 		},
+		{
+			name:      "register0_ab_float32_shift",
+			address:   []uint16{0},
+			quantity:  1,
+			byteOrder: "AB",
+			dataType:  "FLOAT32",
+			shift:     10.,
```

Comment:
Please be expressive
```suggestion
			shift:     10.0,
```